modules/installation-and-upgrade/pages/containerized-proxy-wo-internet.adoc

```diff
@@ -0,0 +1,70 @@
+[[containerized-proxy-setup-without-internet]]
+= Containerized proxy deployment without an internet connection
+
+It is possible to deploy containerized {productname} Proxy in an environment without an internet connection.
+In such case, the images are copied to an internal registry, or saved to a tar file.
+
+This procedure allows re-use of the existing minion FQDN to create a configuration to start a containerized proxy.
+The user should first register the minion to suse manager server, and then re-use the same FQDN in a generation.
+
+
+Make sure the default port is set up to `8022` for virtualized proxy when creating configuration using [literal]`spacecmd`.
```

Comment:
this is not related to connected setup, but rater related to general installation.